### ISSUE TITLE
How to detect iPhone on Unity Console on Windows?

### ISSUE THREAD

**Issue by off99555** [Unity-Technologies/facial-ar-remote#28]
@jonathan-unity Can I use Unity Windows instead of Mac?
I built the Arkit Remote scene using VMware macOS and was able to connect to iPhone in Console.
The frame rate was very low because virtual machine is supposed to be slow.

So I want to run the remote app and connect to Unity on Windows.
But when I open Unity on Windows to try to connect to iPhone, I don't find the iPhone device in the Console list.
![image](https://user-images.githubusercontent.com/15215732/58114294-aecdd200-7c21-11e9-9265-13a01e45093f.png)

How do I detect iPhone on Unity Console on Windows? (I see the iPhone in Windows' iTunes fine)

**Comment by off99555** [comment]
It's already detected now with some magic.
The problem now is that it is laggy like on the virtula macOS.

**Comment by mtschoen-unity** [comment]
Hm.. I'm confused about your issue. Are you trying to connect to the device using the Player Connection like with the BitBucket ARKit plugin? If so, this remote works a little differently. You don't need to connect to the phone from the editor this way. Instead, open the example scene and press Play. You will see your computer's IP address printed to the console, and you can then type that IP into the remote app and press connect. I'm not sure this will work if the editor is running in a virtual machine, but you can connect the remote running on iOS to a Windows computer running the editor. Remember that on Windows, you will have to disable your firewall, or add an exception for Unity over port 9000 to connect to the remote.

**Comment by off99555** [comment]
@mtschoen-unity Oh. I thought the remote from BitBucket is the app I have to install and it will be usable with `facial-ar-remote` as Unity app I tested the remote app with a sample ARKit scene in that project that shows a cube floating in the air.

If you mean that this repo has another remote app that I have to install then let me try and will let you know if it works. Also about the Firewall, is that really necessary?

**Comment by off99555** [comment]
I tried it. I built the iOS of this repo's remote and tap on Connect but nothing happen. It seems like the connect button does not do anything. The button looks like it's disable because there is no animation on the button when tapped. I copied the IP address to put into the phone from Unity Editor on Windows. Windows Defender Firewall is also disabled.
What could be the issue it's not connecting? How could I debug this?
